indico/modules/events/registration/templates/management/reglist_filter.html

```diff
@@ -0,0 +1,55 @@
+{% from 'forms/_form.html' import form_header, form_footer, form_rows %}
+
+{% macro _render_regform_item_col_selector(section) %}
+    <h3>{{ section.title }}</h3>
+    {% for child_item in section.children if child_item.is_field %}
+        {% set has_dropdown = child_item.input_type in ['radio', 'country'] %}
+        <div class="i-label reglist-column js-regform-item {%- if has_dropdown %} filter{% endif %}"
+             data-id="{{ child_item.id }}">
+            <span class="title {% if has_dropdown %}icon-filter{% endif %}"
+                  data-toggle="dropdown">
+                {{- child_item.title -}}
+            </span>
+            {% if has_dropdown %}
+                <ul class="dropdown" style="display: none">
+                    {% for id, caption in child_item.versioned_radioitems.iteritems() %}
+                        <li>
+                            <input type='checkbox' name='radiogroup_{{ child_item.id }}'
+                                   value='{{ id }}'
+                                   id="{{ id }}"
+                                   {% if id in filters %}
+                                       checked
+                                   {% endif %}>
+                            <label for="{{ id }}">
+                                {{ caption }}
+                            </label>
+                        </li>
+                    {% endfor %}
+                </ul>
+            {% endif %}
+            <span class="right actions">
+                <span class="trigger icon-eye"></span>
+            </span>
+        </div>
+    {% endfor %}
+{% endmacro %}
+
+<form class="reglist-filter" method="POST">
+    {% set visible_cols_regform_items = visible_cols_regform_items | tojson | forceescape %}
+    <input id="visible-columns-reg-items" type="hidden" name="visible_cols_regform_items"
+           value="{{ visible_cols_regform_items }}">
+    {% for section in regform.sections %}
+        {{ _render_regform_item_col_selector(section) }}
+    {% endfor %}
+    <div class="bottom-buttons">
+        <input class="i-button big highlight" type="submit" data-disabled-until-change
+               value="{% trans %}Apply{% endtrans %}">
+        <button class="i-button big js-reset-btn" type="button" title="{% trans %}Clear all filters{% endtrans %}">
+            {%- trans %}Clear{% endtrans -%}
+        </button>
+        <button class="i-button big" type="button" data-button-back>{% trans %}Cancel{% endtrans %}</button>
+    </div>
+</form>
+<script>
+    setupRegistrationsListFilter();
```

Comment:
registration.js is used on many pages. At least I don't like calling tons of code you don't use on a certain page. Also, like this you immediately know which code is used on the page.